From 444b905bad6c99757df5673046f015f0f2498690 Mon Sep 17 00:00:00 2001
From: Christian Hewitt <christianshewitt@gmail.com>
Date: Mon, 7 Dec 2020 05:46:45 +0000
Subject: [PATCH 74/97] WIP: arm64: dts: meson: add reboot node to g12-common
 dtsi

This is copied from [0] but adapted to apply to all G12/SM1 boards which
inherit from the meson-g12-common dtsi.

[0] https://github.com/tobetter/linux/commit/ec5bb7ddd6e52025e690b51a6874c875fe4c7c37

Signed-off-by: Christian Hewitt <christianshewitt@gmail.com>
---
 arch/arm64/boot/dts/amlogic/meson-g12-common.dtsi | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/arch/arm64/boot/dts/amlogic/meson-g12-common.dtsi b/arch/arm64/boot/dts/amlogic/meson-g12-common.dtsi
index 9f79f594c6a7..fd4f4a4b0e3b 100644
--- a/arch/arm64/boot/dts/amlogic/meson-g12-common.dtsi
+++ b/arch/arm64/boot/dts/amlogic/meson-g12-common.dtsi
@@ -90,6 +90,12 @@
 		method = "smc";
 	};
 
+	reboot {
+		compatible = "meson,reboot";
+		sys_reset = <0x84000009>;
+		sys_poweroff = <0x84000008>;
+	};
+
 	reserved-memory {
 		#address-cells = <2>;
 		#size-cells = <2>;
-- 
2.17.1

